server/routes/gitlab/gitlab_routes.py

```
@@ -0,0 +1,287 @@
+                params={
+                    "membership": "true",
+                    "order_by": "last_activity_at",
+                    "sort": "desc",
+                    "per_page": per_page,
+                    "page": page,
+                    "simple": "true",
+                },
+                timeout=GITLAB_TIMEOUT,
+            )
+            if resp.status_code != 200:
+                logger.error(f"GitLab API error fetching projects: {resp.status_code}")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)